paddle/fluid/inference/tensorrt/convert/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+nv_library(tensorrt_convert SRCS convert.cc mul_op.cc conv2d_op.cc DEPS dynload_cuda)
+nv_test(test_tensorrt_convert SRCS test_convert.cc DEPS tensorrt paddle_fluid)
```

Comment:
`paddle_fluid` is quite a large dependency, and quite slow for compiling this test.

Just a small issue.